vendor/github.com/mongodb/mongo-tools-common/db/write_concern.go

```diff
@@ -40,10 +40,9 @@ type WriteConcernOptions struct {
 	WTimeout   time.Duration
```

Comment:
the changes here and in `db/db.go` are just re-vendored in. the other changes in `vendor` are parts of this PR.